src/components/date_picker/super_date_picker/time_options.tsx

```diff
@@ -129,13 +130,21 @@ export const useI18nTimeOptions = () => {
     )
     .map((timeUnit) => ({ value: timeUnit, text: timeUnitsPlural[timeUnit] }));
 
+  const refreshUnitsShorthand = {
+    s: useEuiI18n('euiTimeOptions.secondsShorthand', 's'),
+    m: useEuiI18n('euiTimeOptions.monthsShorthand', 'm'),
+    h: useEuiI18n('euiTimeOptions.monthsShorthand', 'h'),
```

Comment:
just noticed I f'd this up while copy-pasting
```suggestion
    h: useEuiI18n('euiTimeOptions.hoursShorthand', 'h'),
```